float32 pitch
---
bool success